core/View.php

```diff
@@ -266,8 +266,7 @@ public function render()
 
             $this->loginModule = Piwik::getLoginPluginName();
 
-            $user = APIUsersManager::getInstance()->getUser($this->userLogin);
-            $this->userAlias = $user['alias'];
+            $this->userAlias = $this->userLogin; // can be removed in Matomo 4.0
```

Comment:
We're not using the alias anymore, and we will even remove it in Matomo 4. Couldn't find any usage in our plugins of this...

Before we were requesting the `user` DB on every VIEW render. Especially when we render the visitor log we might have hundreds or thousands of VIEW renders saving hundreds or thousands of calls to the `user ` DB